api/app/db/VersionsDao.scala

```diff
@@ -356,17 +356,24 @@ class VersionsDao @Inject() (
         val versionName = version.version
         val versionGuid = version.guid
 
-        Logger.info(s"Migrating $orgKey/$applicationKey/$versionName versionGuid[versionGuid] to latest API Builder spec version[$ServiceVersionNumber]")
+        val organizationNamespace = {
+          SQL("select namespace from organizations where guid = {orgGuid}::uuid")
+            .on("orgGuid" -> version.organization.guid)
+            .executeQuery()
+            .as(SqlParser.scalar[String].single)
+        }
```

Comment:
What does `. single` return if there is no record? Might be better to use `OrganizationsDao.findByGuid` and deal with the fact that the org may not exist.

Reply:
Thanks! Addressed.